tox.ini

```diff
@@ -16,12 +16,13 @@ deps =
    isort
    mccabe
    pytest
+   pytest-xdist
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
-    python -Wi {envsitepackagesdir}/coverage run -m pytest {envsitepackagesdir}/pylint/test/ {posargs:}
+    python -Wi {envsitepackagesdir}/coverage run -m pytest -n auto {envsitepackagesdir}/pylint/test/ {posargs:}
```

Comment:
I would prefer to keep ``tox.ini`` untouched, but to provide the flag to pytest through ``posargs``, in ``travis.yml``.